iapyx/src/bin/interactive/mod.rs

```diff
@@ -115,8 +115,8 @@ impl CliController {
         })
     }
 
-    pub fn proposals(&self) -> Result<Vec<Proposal>, Error> {
-        self.backend_client.proposals().map_err(Into::into)
+    pub fn proposals(&self, group: &str) -> Result<Vec<FullProposalInfo>, Error> {
```

Comment:
why this change?

Reply:
I'm not sure I understand the question

Reply:
Why does this return `FullProposalInfo` instead of `Proposal`?

Reply:
Well, `Proposal` doesn't have the voteplan+index now, and this is used to cast votes.